examples/ufo_toy.rs

```diff
@@ -0,0 +1,164 @@
+// Copyright 2019 the Runebender authors.
```

Comment:
This doesn't have to be off in examples. It is possible to have more than one `[[bin]]` in the Cargo.toml, which might be a good way to organize things if we end up having "a bunch of small tools."